pkg/relayer/cmd/cmd.go

```diff
@@ -306,6 +311,24 @@ func newSupplyRelayerProxyFn(
 		deps depinject.Config,
 		_ *cobra.Command,
 	) (depinject.Config, error) {
+		var (
+			queryClientCtx relayer.QueryClientContext
+			keyring        keyring.Keyring
+		)
+		if err := depinject.Inject(deps, &queryClientCtx); err != nil {
+			return nil, err
+		}
+
+		clientCtx := cosmosclient.Context(queryClientCtx)
+
+		deps = depinject.Configs(deps, depinject.Supply(
+			accounttypes.NewQueryClient(clientCtx),
+			suppliertypes.NewQueryClient(clientCtx),
+			sessiontypes.NewQueryClient(clientCtx),
+			apptypes.NewQueryClient(clientCtx),
```

Comment:
I think we should be using this pattern for these types as seen https://github.com/pokt-network/poktroll/pull/187/files#diff-74521651895cccd4d3e3a00792bc7eac38c0d396e86276d226a68c88381f1561:~:text=%40%40%20%2D0%2C0%20%2B1%2C57%20%40%40-,package%20query,-import%20( like we discussed off line.

And the depinject supplier functions like these https://github.com/pokt-network/poktroll/pull/187/files#diff-b0f30e8952edb9a9885d46fa8e6a353fe491fa4d55ddb9104539832599fcdca6:~:text=func%20NewSupplyAccountQuerierFn( which will go into the `cmd.go` supply config section